e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -492,7 +492,7 @@ describe('Plugin Marketplace', () => {
             cy.get('#error_bar').should('not.exist');
         });
 
-        it('display installed plugins', () => {
+        it('MM-T1985 display installed plugins', () => {
```

Comment:
```suggestion
        it('MM-T1979 display installed plugins', () => {
```
Don't this the 'local plugin' reference below really makes sense for this once, hence requesting a change to T1979 instead.